01-introduction.Rmd

```diff
@@ -2,13 +2,15 @@
 
 # Introduction {#c01-intro}
 
+<!-- author review -->
+
 ```{r}
 #| label: globaloptions
 #| include: FALSE
 options(width=72)
 ```
 
-Surveys of random samples are collected to gather information and provide inference to populations. Initially, statisticians argued to obtain purposefully representative samplesk but the field of probability samples grew over time. This book discusses the last part of the survey process, the analysis. We assume that you have survey data that has already been collected and weighted, with the most common scenario being that you are using a public use microdata file. In this book, we will discuss descriptive analysis, statistical testing, and modeling, as well as some best practices in coding and how to present results. We use real data and provide realistic examples. We wanted a text available for people who are new to survey analysis but might already have some statistics and R background.
+Surveys of random samples are collected to gather information and provide inference to populations. Initially, statisticians argued to obtain purposefully representative samples but the field of probability samples grew over time. This book discusses the last part of the survey process, the analysis. We assume that you have survey data that has already been collected and weighted, with the most common scenario being that you are using a public use microdata file. In this book, we will discuss descriptive analysis, statistical testing, and modeling, as well as some best practices in coding and how to present results. We use real data and provide realistic examples. We wanted a text available for people who are new to survey analysis but might already have some statistics and R background.
 
 Most survey microdata distributed publicly includes analysis weights and design variables. These variables are included with the data to correctly calculate estimates. Using these weights is necessary for results to be unbiased. Accounting for the sampling design is also required to calculate correct variance estimates and test statistics. However, as we will discuss in Chapter \@ref(c03-specifying-sample-designs), these calculations can be complex. Several general-purpose statistical software have functions to correctly account for these features in analysis, including SAS, Stata, SUDAAN, and R. This book will use R and a combination of both the {survey} and {srvyr} packages and is meant for people who already have experience in R, namely in using the tidyverse.
```

Comment:
Should we do a callout box for "microdata" and define it?

Should we flush out the description of "Accounting for the sampling design..." sentence? Add in more information about why it is important to calculate variance estimates?

Reply:
+1, I agree with adding some definitions:
* "survey microdata"
* "analysis weights"
* "design variables"
* "sampling design"